ClangSharpPInvokeGenerator/Properties/launchSettings.json

```diff
@@ -1,12 +1,8 @@
-    },
-    "Regenerate ClangSharp - Unix": {
-      "commandName": "Project",
-      "commandLineArgs": "--m clang --p clang_ --namespace ClangSharp --output ../../../../ClangSharp/Generated.cs --libraryPath $(LibClangName) --include $(LLVMIncludePath) --file $(LLVMIncludePath)/clang-c/Index.h --file $(LLVMIncludePath)/clang-c/CXString.h --file $(LLVMIncludePath)/clang-c/Documentation.h --file $(LLVMIncludePath)/clang-c/CXErrorCode.h --file $(LLVMIncludePath)/clang-c/BuildSystem.h --file $(LLVMIncludePath)/clang-c/CXCompilationDatabase.h --e clang_index_getClientEntity --e clang_index_setClientEntity --e clang_createTranslationUnitFromSourceFile --e clang_parseTranslationUnit --e clang_parseTranslationUnit2 --e clang_parseTranslationUnit2FullArgv --e clang_reparseTranslationUnit --e clang_codeCompleteAt --e clang_indexSourceFile --e clang_indexSourceFileFullArgv --e clang_disposeSourceRangeList --e clang_annotateTokens --e clang_annotateTokens --e clang_disposeTokens --e clang_tokenize --e clang_disposeStringSet --e clang_getCompletionParent --e clang_disposeCXPlatformAvailability --e clang_getCursorPlatformAvailability",
+      "commandLineArgs": "--m clang --p clang_ --namespace ClangSharp --output \"../../../../ClangSharp/Generated.cs\" --libraryPath \"$(LibClangName)\" --include \"$(LLVMIncludePath)\" --file \"$(LLVMIncludePath)/clang-c/Index.h\" --file \"$(LLVMIncludePath)/clang-c/CXString.h\" --file \"$(LLVMIncludePath)/clang-c/Documentation.h\" --file \"$(LLVMIncludePath)/clang-c/CXErrorCode.h\" --file \"$(LLVMIncludePath)/clang-c/BuildSystem.h\" --file \"$(LLVMIncludePath)/clang-c/CXCompilationDatabase.h\" --e clang_index_getClientEntity --e clang_index_setClientEntity --e clang_createTranslationUnitFromSourceFile --e clang_parseTranslationUnit --e clang_parseTranslationUnit2 --e clang_parseTranslationUnit2FullArgv --e clang_reparseTranslationUnit --e clang_codeCompleteAt --e clang_indexSourceFile --e clang_indexSourceFileFullArgv --e clang_disposeSourceRangeList --e clang_annotateTokens --e clang_annotateTokens --e clang_disposeTokens --e clang_tokenize --e clang_disposeStringSet --e clang_getCompletionParent --e clang_disposeCXPlatformAvailability --e clang_getCursorPlatformAvailability"
```

Comment:
It might be better for short aliases to just be a single `-` (e.g. `-m` rather than `--m`). This can also be simplified to just `-e function1 function2 function3` (i.e. space delimitation works)